recipes/dirac/meta.yaml

```diff
@@ -0,0 +1,85 @@
+{% set name = "DIRAC" %}
+{% set version = "6.21.5" %}
+{% set major, minor, patch = version.split(".") %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/DIRACGrid/{{ name }}/archive/v{{ major }}r{{ minor }}p{{ patch }}.tar.gz
+  sha256: c032b9bd5f443528c6411265ffebd17755160a9ac3f7ee4d593196fde5cb3243
+  patches:
+    - 0001-Patch-rootPath-to-be-CONDA_PREFIX.patch
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python =2.7
+    - pip
+  run:
+    - python =2.7
```

Comment:
I hope to simplify these at some point but it's not easy to know what this actually needs. I'll fix it in the feedstock once I figure out how to do a better job of testing.